src/payment-providers/tsys/tsys.ts

```diff
@@ -93,6 +93,10 @@ function fetchTsysData(){
     });
 }
 
+function removeAppendChild(code: string){
```

Comment:
This seems a more general regular expressions than I was anticipating, namely since I'm more concerned that it not remove more than is necessary than I am with it not removing enough. 

What do you think about this http://regexr.com/3fhf7 ?